diff --git a/node_modules/babel-plugin-remove-graphql-queries/index.js b/node_modules/babel-plugin-remove-graphql-queries/index.js
index f6ddc07..c8fa190 100644
--- a/node_modules/babel-plugin-remove-graphql-queries/index.js
+++ b/node_modules/babel-plugin-remove-graphql-queries/index.js
@@ -175,12 +175,15 @@ function _default({
       Program(path, state) {
         const nestedJSXVistor = {
           JSXIdentifier(path2) {
-            if ((process.env.NODE_ENV === `test` || state.opts.stage === `develop-html`) && path2.isJSXIdentifier({
+            if ((process.env.NODE_ENV === `test` ||
+              process.env.npm_lifecycle_script?.includes(`storybook`) ||
+              state.opts.stage === `develop-html`) && path2.isJSXIdentifier({
               name: `StaticQuery`
             }) && path2.referencesImport(`gatsby`, ``) && path2.parent.type !== `JSXClosingElement`) {
               const identifier = t.identifier(`staticQueryData`);
               const filename = state.file.opts.filename;
-              const staticQueryDir = state.opts.staticQueryDir || `static/d`;
+              const staticQueryDir =
+                state.opts.staticQueryDir || `page-data/sq/d`
               const shortResultPath = `public/${staticQueryDir}/${this.queryHash}.json`;
               const resultPath = _path.default.join(process.cwd(), shortResultPath);
               // Add query
